revolv/static/css/screen.css

```diff
@@ -4927,11 +5100,12 @@ header.edit-project-header > .container > h1 {
   .project-updates-module .container {
     padding: 38px 39px 20px 92px;
     position: relative;
+    width: 100%;
   }
   /* .project-updates-module .main-area */
-  .project-updates-module .main-area {
+ /* .project-updates-module .main-area {
```

Comment:
Code that is commented out does not serve a purpose. If this is unnecessary, please delete it.